.babelrc

```diff
@@ -1,5 +1,4 @@
 {
-    "presets": [
-        "env", "react"
-    ]
-}
\ No newline at end of file
+  "presets": ["env", "react"],
+  "plugins": ["transform-object-rest-spread"]
```

Comment:
You can use "stage-0" in presets instead of adding more plugins for babel

Reply:
fixed